src/scss/_themes.scss

```diff
@@ -375,3 +375,25 @@
 		background-size: 35px;
 	}
 }
+
+@mixin oTeaserVideo {
+	.o-teaser__image-container {
+		position: relative;
+
+		&:after {
+			@include oIconsGetIcon('play', oColorsGetPaletteColor('white'), $apply-width-height: false);
+			content: '';
+			position: absolute;
+			bottom: 0;
+			left: 0;
+			background-color: oColorsGetPaletteColor('black');
+			width: 15%;
+			height: 15% * (16 / 9);
```

Comment:
Just wondering then if it would be possible to do the padding percentage hack using `padding-top:100%` (I think) and 0 height, then pos absolute the icon? It's probably the only way of preventing it from relying on a 16:9 image. I'd only consider it because in the top stories slice, it can have a square image, and if they want that to be a video it'll break there...